funnel/util.py

```diff
@@ -1,8 +1,11 @@
-from . import app
+import StringIO
```

Comment:
Usually we do `from StringIO import StringIO` because there's nothing else we need from the class. Also, in some cases this construct helps:

```python
try:
    from cStringIO import StringIO
except ImportError:
    from StringIO import StringIO
```

If you're dealing with potentially large data, cStringIO is faster, but won't work under pypy so you need the fallback.